map/src/main/res/layout/minimal_exo_playback_control_view.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="50dp"
+    android:background="@color/greyText">
+
+    <FrameLayout
+        android:id="@+id/playPause"
+        android:layout_width="32dp"
+        android:layout_height="32dp"
+        android:layout_marginStart="@dimen/marginStandard"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ImageButton
+            android:id="@id/exo_play"
+            android:layout_width="32dp"
+            android:layout_height="32dp"
+            android:background="@drawable/ic_play" />
+
+        <ImageButton
+            android:id="@id/exo_pause"
+            android:layout_width="32dp"
+            android:layout_height="32dp"
+            android:background="@drawable/ic_pause" />
+    </FrameLayout>
+
+    <TextView
+        android:id="@+id/trackTitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/marginStandard"
+        android:fontFamily="@font/ideal_sans_book"
+        android:padding="4dp"
+        android:textColor="@color/white"
+        android:textSize="@dimen/textMedium"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/closePlayer"
+        app:layout_constraintStart_toEndOf="@id/playPause"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Test track" />
+
+    <ImageButton
+        android:id="@+id/closePlayer"
+        android:layout_width="32dp"
```

Comment:
do we need to set image sizes. can we get away with `wrap_content` on these? the images should contain the intrinsic size already, no?